howso/ablation.amlg

```diff
@@ -139,6 +139,77 @@
 		(call !CreateCaseWeights (assoc feature_name weight_feature) )
 	)
 
+	;reduce a model by removing cases which have an influence weight entropy that falls above
+	; a threshold
+	;
+	;parameters:
+	; context_features: a list of context features to use when computing influence weight entropies
+	; distribute_weight_feature: optional, default '.case_weight'. name of feature whose values to use as case weights
+	; influence_weight_entropy_threshold: optional, default 0.6. quartile to use as the cutoff threshold for
+	;influence weight entropy when deciding which cases to remove.
+	#reduce_model
+	(declare
+		(assoc
+			context_features (list)
+			distribute_weight_feature (null)
+			influence_weight_entropy_threshold (null)
+
+			;internal
+			max_influence_weight_entropy_to_keep .infinity
+		)
+
+		(assign (assoc
+			distribute_weight_feature
+				(if
```

Comment:
for simple if statements like these two, please put the condition on the same line as the (if statement.